mgmt/api/INKMgmtAPI.cc

```diff
@@ -1865,8 +1865,7 @@ TSGetErrorMessage(TSMgmtError err_id)
 tsapi TSMgmtError
 TSEncryptPassword(char *passwd, char **e_passwd)
```

Comment:
No we don't need to do API review for the management API, only for plugin APIs.

> On Jun 2, 2016, at 9:37 AM, Masaori Koshiba <notifications@github.com> wrote:
> 
> In mgmt/api/INKMgmtAPI.cc:
> 
> > @@ -1865,8 +1865,7 @@ TSGetErrorMessage(TSMgmtError err_id)
> >  tsapi TSMgmtError
> >  TSEncryptPassword(char *passwd, char **e_passwd)
> I agree with that. Do we need API review to remove? This looks like one of TS API.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>